neuralpredictors/layers/shifters/static_affine.py

```diff
@@ -0,0 +1,59 @@
+from .base import Shifter
+
+import torch
+from torch import nn
+from torch.nn import ModuleDict
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class StaticAffine2d(Shifter):
```

Comment:
I don't think this would work - perhaps I mistakingly replaced the superclass here. I'm pretty sure this was meant to inherit from `nn.Linear`